.vscode/settings.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Usually IDE settings are not sent in the repository.

Reply:
Yup, but we agreed (?) to recommend VSCode that's why I've included such settings with extensions recommendations too 👍 

Let me know if settings are too much and if I should remove extensions recommendations too